labscript_utils/__init__.py

```diff
@@ -15,6 +15,7 @@
 import os
 import traceback
 from pathlib import Path
+import importlib
 
 from .versions import get_version, NoVersionInfo
```

Comment:
`get_version` redefined below (line 59, where unused `VersionException` is imported).